modules/menu/css/stories.tsx

```diff
@@ -1,22 +1,71 @@
-import React from 'react';
+import React, {Component} from 'react';
 import {storiesOf} from '@storybook/react';
 import withReadme from 'storybook-readme/with-readme';
 import README from './README.md';
-import {css} from 'emotion';
 import './index.scss';
+// @ts-ignore
+import initializeIcons from '../../icon/css/lib/canvas-kit-css-icon';
```

Comment:
I had to add a `@ts-ignore` directive here to prevent a `TS7016: Could not find a declaration file for module...` error (copied this from the CSS Icon stories). I hope this is ok?